pkg/uprobetracer/tracer.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package uprobetracer
+
+import (
+	"fmt"
+	"github.com/cilium/ebpf"
+	containercollection "github.com/inspektor-gadget/inspektor-gadget/pkg/container-collection"
+	"sync"
+)
+
+type inode_uuid struct {
+	fs_id int
```

Comment:
```suggestion
	fsID unix.Fsid
```

Type of fsID:
https://pkg.go.dev/golang.org/x/sys@v0.18.0/unix#Fsid